admin/client/App/screens/Item/components/RelatedItemsList.js

```diff
@@ -22,7 +23,12 @@ const RelatedItemsList = React.createClass({
 	getColumns () {
 		const { relationship, refList } = this.props;
 		const columns = refList.expandColumns(refList.defaultColumns);
-		return columns.filter(i => i.path !== relationship.refPath);
+		const refListNameColumn = columns.filter(column => column.path === refList.namePath || (column.path === 'id' && refList.namePath === '_id'));
+		const refListRelationshipFieldColumn = columns.filter(column => column.path === relationship.refPath);
+		return [refListNameColumn[0], refListRelationshipFieldColumn[0]];
```

Comment:
@wmertens probably an issue in the original code as well?  I can look into this, but I'm hesitant to spend any more time on this if @JedWatson is not going to merge it :=)